content/en/synthetics/uptime_check.md

```diff
@@ -141,6 +141,24 @@ A check is considered `FAILED` if it doesn't satisfy the assertions configured f
 | UNABLE_TO_GET_ISSUER_CERT_LOCALLY  | The issuer certificate of a locally found certificate is not found. This usually means that the list of trusted certificates is not complete.                         |
 | UNABLE_TO_VERIFY_LEAF_SIGNATURE    | No signatures are verified because the certificate chain contains only one certificate, which is not self-signed, and the issuer is not trusted.                      |
 
+## External metrics
+
+The following metrics are generated by uptime checks.
+
+| | |
+| --- | --- |
+| **synthetics.http.response.time** <br/> (gauge) | Overall time needed for the request to be processed. This is an independent metric; it is not computed as a sum of all the different network timing metrics (DNS, SSL, download, etc.) A warning appears in check runner logs if this value is too different from the sum of all network timing metrics. <br/> _shown as milliseconds_ |
+| **synthetics.http.response.size** <br/> (gauge) | Size of the body. Only computed when body is returned. <br/> _shown as bytes_ |
+| **synthetics.http.redirect.time** <br/> (gauge) | Duration of redirects before the last request.  <br/> _shown as milliseconds_ |
+| **synthetics.http.dns.time** <br/> (gauge) | Time spent resolving the DNS name of the last request. <br/> _shown as milliseconds_ |
+| **synthetics.http.connect.time** <br/> (gauge) | Time spent establishing a connection to the server; duration of the TCP connection. <br/> _shown as milliseconds_ |
+| **synthetics.http.ssl.time** <br/> (gauge) | Time spent for the TLS handshake. If the last request is not over HTTPS, this metric does not appear. <br/> _shown as milliseconds_ |
+| **synthetics.http.firstbyte.time** <br/> (gauge) | Time spent waiting for the first byte of response to be received. <br/> _shown as milliseconds_ |
+| **synthetics.http.download.size** <br/> (gauge) | Time spent downloading the response. <br/> _shown as milliseconds_ |
```

Comment:
It should be `synthetics.http.download.time` rather than `synthetics.http.download.size`